lib/lens.ex

```diff
@@ -427,6 +427,35 @@ defmodule Lens do
     lenses |> Enum.reverse() |> Enum.reduce(empty(), &both/2)
   end
 
+  @doc ~S"""
+  Returns a lens that contains what the first lense focuses on, unless it result is empty. In the latter case the
```

Comment:
lense -> lens
contains -> focuses on
it result is empty -> the first lens focuses on nothing